src/Umbraco.Examine/MediaValueSetBuilder.cs

```diff
@@ -29,6 +32,32 @@ public override IEnumerable<ValueSet> GetValueSets(params IMedia[] media)
             foreach (var m in media)
             {
                 var urlValue = m.GetUrlSegment(_urlSegmentProviders);
+
+                var umbracoFilePath = string.Empty;
+                var umbracoFile = string.Empty;
+
+                var umbracoFileSource  = m.GetValue<string>(Constants.Conventions.Media.File);
+
+                if (umbracoFileSource.DetectIsJson())
+                {
+                    var cropper = JsonConvert.DeserializeObject<ImageCropperValue>(m.GetValue<string>(Constants.Conventions.Media.File));
```

Comment:
We'll need a try/catch here just in case since if this fails then all indexing fails

Reply:
Done, to log the exception I needed to inject the logger which of course meant I had to mock the logger for the tests (please check)